bofire/utils/cheminformatics.py

```diff
@@ -0,0 +1,94 @@
+import warnings
+from typing import List
+
+import numpy as np
+
+try:
+    from rdkit.Chem import AllChem, Descriptors, MolFromSmiles  # type: ignore
+except ImportError:
+    warnings.warn(
+        "rdkit not installed, BoFire's cheminformatics utilities cannot be used."
+    )
+from sklearn.feature_extraction.text import CountVectorizer
+
+# This code is based on GAUCHE: https://github.com/leojklarner/gauche/blob/main/gauche/data_featuriser/featurisation.py
+
```

Comment:
what about all the backward transformations? Mol2smiles etc

Reply:
Not yet in the focus.